modules/publicize/publicize-jetpack.php

```diff
@@ -447,7 +447,11 @@ function test_connection( $service_name, $connection ) {
 	 * Save a flag locally to indicate that this post has already been Publicized via the selected
 	 * connections.
 	 */
-	function save_publicized( $post_ID, $post, $update ) {
+	function save_publicized( $post_ID = null, $post = null, $update = null ) {
```

Comment:
The plugin that causes this error does the following. 
do_action( 'wp_insert_post', 'wp_insert_post'); 

It is true that I don't need to check all the values. I could just check that $post_id is a number. 

The reason why I implemented it this way was to attempt to make it more future proof. 
What if a plugin comes along and does something that we don't expect. Like ignore the $update value or not set the post or just call `do_action( 'wp_insert_post' ); `

Even though the plugin does something unexpected Jetpack should try to prevent PHP errors from showing up.